src/knn.jl

```diff
@@ -56,17 +56,17 @@ function knn(tree::NNTree{V}, point::Matrix{T}, k::Int, sortres=false, skip::Fun
     knn(tree, new_data, k, sortres, skip)
 end
 
-function nn(tree::NNTree{V}, points::Vector{T}, sortres=false, skip::Function=always_false) where {V, T <: AbstractVector}
-    idx, dist = knn(tree, points, 1, sortres, skip)
+function nn(tree::NNTree{V}, points::Vector{T}, skip::Function=always_false) where {V, T <: AbstractVector}
+    idx, dist = knn(tree, points, 1, skip)
```

Comment:
You still have to put `false` here for `sortres` as `knn` needs it.

Reply:
I don't think so. `knn` provides the default of false.

Reply:
But the order of arguments is important. The test suite fails because the `skip` argument is being read as the `sortres` argument and then there is a boolean interpretation error.

Reply:
Ah, confused keyword args and successive optional args. Thanks.